jdl/jhipster/reserved-keywords.js

```diff
@@ -66,11 +67,11 @@ function isReservedTableName(keyword, databaseType) {
 
 function isReservedFieldName(keyword, clientFramework) {
     if (clientFramework) {
-        if (clientFramework === 'angularX') {
+        if (clientFramework === applicationOptions.OptionValues[applicationOptions.OptionNames.CLIENT_FRAMEWORK].angularX) {
```

Comment:
Define constant clientFrameworks and use it in conditions, it is more readable.
const clientFrameworks = applicationOptions.OptionValues[applicationOptions.OptionNames.CLIENT_FRAMEWORK];